schemas/team/summary.yml

```diff
@@ -0,0 +1,44 @@
+type: object
+
+properties:
+  id:
+    type: string
+    readOnly: true
+    description: |
+      The team's identifier (generated by the API)
+
+  url:
+    type: string
+    readOnly: true
+    description: |
+      API url for accessing the team description
+
+  title:
+    type: string
+    readOnly: true
+    description: |
+      The human-readable title of the team
+
+  members:
+    type: array
+    readOnly: true
+    items:
+      $ref: '#/definitions/user/summary'
+    description: |
+      Summaries of users that are part of this team.
+
+  organization:
+    $ref: '#/definitions/organization/summary'
+    readOnly: true
+    description: |
+      Summary of the organization that owns this team.
```

Comment:
I vote for changing seed-auth-api and the docs. It probably does mean more joins, not sure if it that is going to complicate things, but I think the less api juggling the UI will need to do the better.